package.json

```diff
@@ -1,12 +1,15 @@
 {
   "name": "@eppo/js-client-sdk-common",
-  "version": "1.6.0",
+  "version": "1.6.1",
```

Comment:
Given our change in the required node version, I wonder if we make this a major version change to `1.7.0` to reflect it may not be backward compatible.

Reply:
Yes absolutely thanks for mentioning.